framework/architecture/businessunit/hierarchical-business-units.rst

```diff
@@ -152,12 +152,12 @@ Technical implementation
 Hierachical business units actually do not have anything to do with the with business unit handling itself. It is more
 something built on top of the business unit.
 
-The read and write permissions are handled by the :java:ref:`ch.tocco.nice2.businessunit.impl.hierarchy.GenericHierarchyPolicyProcessor`.
-This policy processor adds rules (:java:ref:`ch.tocco.nice2.security.Rule`) depending on the contributed
-:java:ref:`ch.tocco.nice2.businessunit.impl.hierarchy.HierarchySpecification` during the login phase.
+The read and write permissions are handled by the :abbr:`ch.tocco.nice2.businessunit.impl.hierarchy.GenericHierarchyPolicyProcessor (GenericHierarchyPolicyProcessor)`.
```

Comment:
:java:ref: wurden falsch ersetzt in diesem File (der volle Klassenname sollte als Popup angzeigt werden, nicht umgekehert)